graylog2-web-interface/src/components/bootstrap/BootstrapModalForm.jsx

```diff
@@ -21,6 +21,7 @@ class BootstrapModalForm extends React.Component {
     onModalClose: () => {},
     onSubmitForm: undefined,
     onCancel: () => {},
+    bsSize: 'large',
```

Comment:
Most modals we use didn't set this property, so I guess we should leave this as `undefined` by default.